core/src/cluster_slots_service.rs

```diff
@@ -119,7 +119,12 @@ impl ClusterSlotsService {
                 }
             }
             let root_bank = bank_forks.read().unwrap().root_bank();
-            cluster_slots.update(&root_bank, &cluster_info);
+            cluster_slots.update(
```

Comment:
Changing the way update is called to reduce internal exposure to the details of RootBank impl